RecoEcal/EgammaCoreTools/test/testEcalClusterLazyTools.cc

```diff
@@ -47,39 +47,31 @@ Description: <one line class summary>
 
 
 
-class testEcalClusterLazyTools : public edm::EDAnalyzer {
+class testEcalClusterLazyTools : public edm::global::EDAnalyzer<> {
```

Comment:
Use of `global` is not really consistent with the use of `std::cout` (latter not being thread-safe). Even with MessageLogger I'd think `one` to be a better choice as then the outputs from different events would be guaranteed to not overlap. (even if this module would "always" be run with a single thread)

Reply:
True, I also realized that after my commit. I'm just too obsessed with global modules.